addons/mail/static/src/scss/activity_view.scss

```diff
@@ -34,6 +34,7 @@
         &:hover {
             color: #666;
         }
+        cursor: pointer;
```

Comment:
so rules are duplicated for each view?

Reply:
Only used for activity view, what's the problem?

Reply:
The same widget is used in the kanban view, so i assume that the same kind of rule is defined elsewhere. Ideally, the widget, its template, and its scss rules should be extracted somewhere, out of the scope of any view, such that it can be easily reused in other contexts (e.g. in the activity view).